Final combined version of web app

Gregor-Davis opened the pull request:
Added all links for nav bar and moved css/js into separate folders

Gregor-Davis committed:
Final combined version of web app

Added all links for nav bar and moved css/js into separate folders

Willsbond reviewed (commented):
Looked through the project. All looking good. Ready to deploy in my opinion. Good work team!

chithepirate commented:
Wait a minute. For some reason the custom font I added to the recipe page is not working. Let me review the files carefully to see what caused this issue...

chithepirate committed:
css issue

there is an unknown issue that causes the @font-face inside the css file to break, so I moved the reference to it inside the html for the recipe page.

chithepirate committed:
new recipe.css

chithepirate committed:
replacing recipes.html with fixed version

chithepirate committed:
Final recipes.html

Willsbond committed:
Presentation

Gregor-Davis committed:
Updated nav bar issue
